tools/cldr/cldr-to-icu/lib/com/ibm/icu/icu-utilities/0.1-SNAPSHOT/icu-utilities-0.1-SNAPSHOT.pom

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.ibm.icu</groupId>
+  <artifactId>icu-utilities</artifactId>
+  <version>0.1-SNAPSHOT</version>
+  <description>POM was created from install:install-file</description>
```

Comment:
neither this (nor the .jar ) should be here in git. As a stop-gap it should be generated and published as part of icu4j's release.   @yumaoka  ?

Reply:
FYI there's now an "api-jar" target in the Ant file which produces "cldr-api.jar" which is a smaller JAR than using "cldr.jar".

Reply:
If you want to go that route (getting things in and iterate), then I would prefer a set of instructions which includes running the mvn install yourself - but still, not including the jars or the mvn repo in the icu repository. As I understand it, for this route the 'cldr-to-icu' tool is not going to be the primary generator for ICU 65's data.

Reply:
Yeah, I was working on the assumption it'd be used until 2 days ago :)